README.md

```diff
@@ -35,6 +35,17 @@ Featured in the video:
 * Showing diagnostic message in a floating window.
 * Code actions.
 
+## How to get the most out of this plugin
+
+This plugin is designed to provide an opinionated and useable set of defaults for working with LSP and Completion in Neovim. A few things should be considered before you use it in your own config
```

Comment:
Here it should say "autocompletion" instead of just "completion".

Thing is neovim already has a completion mechanism, but its very manual.

Reply:
NP, Simple fix, will change in next commit.